pyproject.toml

```diff
@@ -56,6 +52,7 @@ module = [
     "web.blueprints",
     "web.blueprints.access",
     "web.blueprints.facilities",
+#    "web.blueprints.host",
```

Comment:
That should be reverted, but I am lazy and we want to look at enabling stricter rules for most modules, anyway